app/src/main/python/gogdl/saves.py

```
@@ -0,0 +1,365 @@
+            sys.stdout.flush()
+            
+        except Exception as e:
+            self.logger.error(f"Sync failed: {e}")
+            raise
+
+    def get_auth_ids(self):
+        """Get client credentials from auth manager"""
+        try:
+            # Use the same client ID as the main app
+            client_id = "46899977096215655"
+            client_secret = "9d85c43b1482497dbbce61f6e4aa173a433796eeae2ca8c5f6129f2dc4de46d9"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "client_secret"

(S105)